ci/src/main/scala/org/typelevel/sbt/CrossRootProject.scala

```diff
@@ -20,8 +20,8 @@ import sbt._
 import org.typelevel.sbt.gha.GenerativePlugin.autoImport._
 
 /**
- * Simultaneously creates a `root`, `rootJVM`, `rootJS`, and `rootNative` project, and
- * automatically enables the `NoPublishPlugin`.
+ * Simultaneously creates a root project, a Scala JVM subproject, a Scala.js subproject, a Scala
+ * Native subproject and automatically enables the `NoPublishPlugin`.
```

Comment:
I actually feel like this is less clear about what it does :)

Maybe "aggregate project" would be clearer than "sub project"?